staging/src/k8s.io/pod-security-admission/policy/checks.go

```diff
@@ -45,10 +45,15 @@ type VersionedCheck struct {
 	MinimumVersion api.Version
 	// CheckPod determines if the pod is allowed.
 	CheckPod CheckPodFn
+	// OverrideCheckIDs is an optional list of checks that should be skipped when this check is run.
+	// Overrides may only be set on restricted checks, and may only override baseline checks.
+	OverrideCheckIDs []CheckID
```

Comment:
is per-versioned-check overrides exercised in tests?

Reply:
Yes:
https://github.com/tallclair/kubernetes/blob/d1383682d8294d36737f8ca3d119d9c3a10d1dc9/staging/src/k8s.io/pod-security-admission/policy/registry_test.go#L31-L40
```
	checks := []Check{
		generateCheck("a", api.LevelBaseline, []string{"v1.0"}),
		generateCheck("b", api.LevelBaseline, []string{"v1.10"}),
		generateCheck("c", api.LevelBaseline, []string{"v1.0", "v1.5", "v1.10"}),
		generateCheck("d", api.LevelBaseline, []string{"v1.11", "v1.15", "v1.20"}),
		generateCheck("e", api.LevelRestricted, []string{"v1.0"}),
		generateCheck("f", api.LevelRestricted, []string{"v1.12", "v1.16", "v1.21"}),
		withOverrides(generateCheck("g", api.LevelRestricted, []string{"v1.10"}), []CheckID{"a"}),
		withOverrides(generateCheck("h", api.LevelRestricted, []string{"v1.0"}), []CheckID{"b"}),
	}
```

1. check `g` (added in v1.10) overrides check `a` (added in v1.0)
2. check `h` (added in v1.0) overrides check `b` (added in v1.10)

Reply:
I actually meant different versions of a single check overriding different ids

```go
[]Check{
  {ID:"a",Level:"baseline",Versions:[]VersionedCheck{{MinimumVersion: "v1.0"}}},
  {ID:"b",Level:"baseline",Versions:[]VersionedCheck{{MinimumVersion: "v1.2"}}},
  {ID:"c",Level:"baseline",Versions:[]VersionedCheck{
    {MinimumVersion: "v1.4",OverrideCheckIDs:[]CheckID{"a"}},
    {MinimumVersion: "v1.6",OverrideCheckIDs:[]CheckID{"b"}},
  }},
}
```

Reply:
Done.